### ISSUE TITLE
cohomolo on Windows

### ISSUE THREAD

**Issue by rtange** [gap-packages/cohomolo#15]
It would be nice if cohomolo were made to work on Windows.

At the moment when I load Cohomolo 1.6.6 on
GAP 4.9.2 of 04-Jul-2018 on Windows 7 and type
```
G:=SymmetricGroup(5);;p:=2;;
chr:=CHR(G,2,0,Z(2)*[[[1]],[[1]]]);;
FirstCohomologyDimension(chr);
```
I get:
```
'rm' is not recognized as an internal or external command,
operable program or batch file.
Error, no method found! For debugging hints type ?Recovery from NoMethodFound
Error, no 1st choice method found for `SetPrintFormattingStatus' on 2 arguments
The 1st argument is 'fail' which might point to an earlier problem
 at /proc/cygdrive/C/gap-4.9.2/lib/methsel2.g:250 called from
SetPrintFormattingStatus( stream, false ); at /proc/cygdrive/C/gap-4.9.2/pkg/cohomolo-1.6.6/gap/coho4.g:57 called from
COHOMOLO.WritePermGroup( G, deg, base, Concatenation( filename, ".inperm" )
 ); at /proc/cygdrive/C/gap-4.9.2/pkg/cohomolo-1.6.6/gap/coho4.g:600 called from
Cohomolo( chr, false, false, true, TmpName(  )
 ); at /proc/cygdrive/C/gap-4.9.2/pkg/cohomolo-1.6.6/gap/coho4.g:899 called from
<function "FirstCohomologyDimension">( <arguments> )
 called from read-eval loop at *stdin*:4
```
In the past it has been noted that cohomolo can work on Windows:
""cohomolo" also runs seemingly OK on Windows with Cygwin"
See https://mail.gap-system.org/pipermail/forum/2004.txt

But I could not make it work with these instructions.

Rudolf Tange

**Comment by fingolfin** [comment]
@ChrisJefferson any idea if this might work now on Windows?

It complains about `rm` missing but that should be fine these days?

**Comment by rtange** [comment]
In GAP 4.10.1 of 23-Feb-2019 (on Windows 10 Enterprise) I still get the same error message as above.
Is this only supposed to be resolved in GAP 4.12.0?

**Comment by fingolfin** [comment]
Might also be resolved in 4.11.1 which I think already had the "new" Windows installer.

Any particular reason you don't want to update to 4.12.0?

**Comment by rtange** [comment]
I installed GAP 4.12.0 via the windows installer and it works now,
but the commmand "FirstCohomologyDimension(chr);" took several seconds
and on a unix machine it give the output immediately.
So I will probably reinstall with WSL.
